kitsune/bundles.py

```diff
@@ -28,8 +28,10 @@
     },
     'mobile-common': {
         'source_filenames': (
+            'fontawesome/css/font-awesome.css',
```

Comment:
FYI this won't actually work on the servers right now because of [bug 1150592](https://bugzilla.mozilla.org/show_bug.cgi?id=1150592).